### ISSUE TITLE
Trait not found in laravel 5.4.34

### ISSUE THREAD

**Issue by deepankarmondal** [spatie/laravel-activitylog#245]
I have followed all the step  for installing the package when i am trying to use it for model log it showing 

Trait 'Spatie\Activitylog\Traits\LogsActivity' not found Error
 My Model Code

<?php

namespace App;

use Illuminate\Database\Eloquent\Model;
use Spatie\Activitylog\Traits\LogsActivity;

class Payments extends Model
{
    use LogsActivity;
    protected $logAttributes  = ['column1', 'column2', 'column3', 'column4', 'column5', 'column6' ];
    protected static $logOnlyDirty = true;
    protected $fillable = ['column1', 'column2', 'column3', 'column4', 'column5', 'column6' ];
}

**Comment by freekmurze** [comment]
Does this file existing in the package directory inside the `vendor` directory?

https://github.com/spatie/laravel-activitylog/blob/master/src/Traits/LogsActivity.php

**Comment by freekmurze** [comment]
Try installing it via Composer in a vanilla project.

**Comment by AlexVanderbist** [comment]
Closing this due to inactivity of OP.

**Comment by atulshrma91** [comment]
Hello,

I'm facing the same issue. I have upgraded laravel 5.1 to laravel 5.5.
I did this - 

Simply comment out the line within the ServiceProvider, publish the migration, rename the class in the new migration file to something like CreateNewActivityLogTable and then run composer dump

But now project I'm getting trait missing.  "Trait 'Spatie\Activitylog\LogsActivity' not found"

LogsActivity.php is found in \vendor\spatie\laravel-activitylog\src\Traits  in my installation. my composer has "spatie/laravel-activitylog": "^2.5" 

I have migrated the new activity log table, manually added content to app/activitylog.php and in service provider array added "Spatie\Activitylog\ActivitylogServiceProvider::class,"
